### PR TITLE
made links

### PR DESCRIPTION
**Pull request by TheLuckyPerson**

### CODE REVIEW

**Comment by rrios07** [comment]
Links looked good, just had to run prettier for formatting.

**Comment by DrewskiG123** [comment]
Richard summed it up well, looks good, just needed some formatting.

File: packages/react-frontend/src/HomePage.js [hunk 1]
```diff
@@ -1,75 +1,69 @@
...
+        <Routes>
+            <Route path="Sign In" element={<SignIn/>} />
+            <Route path="Sign Up" element={props.SignUp} />
+            <Route path="Home" element={props.Home}/>
+            <Route path="" element={props.Home}/>
+            {/* <Route path="Profile" element={<Profile />} /> */}
+            <Route path="Post" element={props.Post} />
+        </Routes>
+    </BrowserRouter>
+</div>
```
Comment (casaresi-calpoly) [inline packages/react-frontend/src/HomePage.js:66]: This all looks really good to me. Would there be a way to redirect a user back to the home page if they every put in an invalid link?

**Review by casaresi-calpoly — COMMENTED** [review]
Overall looks good. As my comment suggest, see if there is a way to redirect a user to the home page if they put in an invalid link

**Pull request merged**